io/disk/arcconf/arcconf_cntl_oper.py.data/arcconf_cntl_oper.yaml

```diff
@@ -0,0 +1,171 @@
+Test: !mux
+   Backupunit:
+        option: "arcconf BACKUPUNIT"
+        option_args: "RESET"
+   Consistencycheck1:
+        option: "arcconf CONSISTENCYCHECK"
+        option_args: "off"
+   Consistencycheck2:
+        option: "arcconf CONSISTENCYCHECK"
+        option_args: "on"
+   Consistencycheck3:
+        option: "echo \"y\" | arcconf CONSISTENCYCHECK"
+        option_args: "PERIOD 10"
+   Copyback1:
+        option: "arcconf COPYBACK"
+        option_args: "OFF"
+   Copyback2:
+        option: "arcconf COPYBACK"
+        option_args: "ON"
+   Errortunable1:
+        option: "arcconf ERRORTUNABLE"
+        option_args: "SETPROFILE 2"
+   Errortunable2:
+        option: "arcconf ERRORTUNABLE"
+        option_args: "CONFIGPARAMS CRC_ERROR_DURATION_SEC 10"
+   Errortunable3:
+        option: "arcconf ERRORTUNABLE"
+        option_args: "GETPARAMS"
+   Errortunable4:
+        option: "echo \"y\" | arcconf ERRORTUNABLE"
+        option_args: "GETPARAMS 2 SAVE /tmp/abc.cfg"
+   Expanderlist:
+        option: "arcconf EXPANDERLIST"
+        option_args: ""
+   Failover1:
+        option: "arcconf FAILOVER"
+        option_args: "off"
+   Failover2:
+        option: "arcconf FAILOVER"
+        option_args: "on"
+   Getexception:
+        option: "arcconf GETEXCEPTION"
+        option_args: "AL"
+   Getlogs:
+        option: "arcconf GETLOGS"
+        option_args: "STATS tabular"
+   Getperform1:
+        option: "arcconf GETPERFORM"
+        option_args: "1"
+   Getperform2:
+        option: "arcconf GETPERFORM"
+        option_args: "1 save /tmp/abc.cfg"
+   Getsmartstats1:
+        option: "arcconf GETSMARTSTATS"
+        option_args: ""
+   Getsmartstats2:
+        option: "arcconf GETSMARTSTATS"
+        option_args: "tabular"
+   Getstatus:
+        option: "arcconf GETSTATUS"
+        option_args: ""
+   Getversion:
+        option: "arcconf GETVERSION"
+        option_args: ""
+   List:
+        option: "arcconf LIST"
+        option_args: ""
+   Saveconfig:
+        option: "arcconf SAVECONFIG"
+        option_args: "/tmp/abc.xml /tmp/abc.log"
```

Comment:
done @narasimhan-v , fixed in v2  .     

it gets deleted in every iteration so i named it as /tmp/arcconf_cntl.log